Makefile

```diff
@@ -10,10 +11,13 @@ endef
 build:
 	cp config/database.yml.example config/database.yml
 	cp config/config.yml.example config/config.yml
+	cp config/initializers/recaptcha.rb.example config/initializers/recaptcha.rb
 	cp db/schema.rb.example db/schema.rb
+	docker-compose down --remove-orphans
 	docker-compose build
 
 deploy-container:
+	docker-compose run --rm web bash -l -c "sleep 10 && bower install --allow-root && rake db:setup && rake db:migrate && rake assets:precompile"
```

Comment:
The `wait_for_container()` method dismisses the need of `...sleep 10...`, in MK we're not using Bower anymore, I remember we've switched to Yarn a while ago. Also, I think those commands such as `rake *` commands are already defined at the `script.sh` file, here -> https://github.com/publiclab/mapknitter/blob/main/start.sh.